Tools/WebKitTestRunner/TestController.cpp

```diff
@@ -1055,6 +1055,7 @@ void TestController::resetPreferencesToConsistentValues(const TestOptions& optio
         if (enableAllExperimentalFeatures) {
             WKPreferencesEnableAllExperimentalFeatures(preferences);
             WKPreferencesSetExperimentalFeatureForKey(preferences, false, toWK("AlternateWebMPlayerEnabled").get());
+            WKPreferencesSetExperimentalFeatureForKey(preferences, false, toWK("SiteIsolationEnabled").get());
```

Comment:
I kind of wonder if doing this should be an indication that your feature belongs in -Internal, not -Experimental, for the time being 🤷